src/annotator/anchoring/test/text-range-test.js

```diff
@@ -468,5 +452,21 @@ describe('annotator/anchoring/text-range', () => {
         assert.equal(textRange.end.offset, 10);
       });
     });
+
+    describe('trimmedRange', () => {
+      it('adjusts Range start and end positions to remove whitespace', () => {
+        const el = document.createElement('div');
+        el.textContent = ' one two three ';
+
+        const range = new Range();
+        range.selectNodeContents(el);
+
+        const textRange = TextRange.fromRange(range).toRange();
+        const trimmedRange = TextRange.trimmedRange(range);
```

Comment:
This test module does not mock `trim-range`. i.e. this is not a unit test. But I'm not sure what the value of a pure unit test would _be_ here; curious as to your thoughts.